### DISCUSSION TITLE
Is there an alternatie command for SetTextGDIPlusProperties available ?

### DISCUSSION THREAD

**Discussion by henry-delaruelle** [Palakis/obs-websocket#1019] [Q&A]
I use the SetTextGDIPlusProperties command to set subtitles by changing the content of a Text(GDI+) source. is there an alternative way of doing this via the new (5.01) protocol ? (only being able to change the text would be fine)

**Comment by tt2468** [comment]
SetInputSettings, and the field to change should be `text`

**Reply by henry-delaruelle** [reply]
Thanks, downloading the settings for inputKind "text_gdiplus_v2", gave me an object i could  use to send the correct parameters

**Comment by notfree** [comment]
hello, 
im trying to change the text in the "News" text item , but nothing happened 
obs.call(requests.SetInputSettings(inputName="News",Text="Test"))

can someone make an object list so i can see how to address it correct ?
